src/web/source.rs

```diff
@@ -193,10 +193,47 @@ pub fn source_browser_handler(req: &mut Request) -> IronResult<Response> {
     let storage = extension!(req, Storage);
     let config = extension!(req, Config);
 
+    // TODO should we use match_version here?
+    let archive_storage: bool = {
```

Comment:
This seems to already work with wildcards, `/stm32f4/*/src/stm32f4/stm32f401/exti.rs.html#1-82` goes to `/stm32f4/0.13.0/src/stm32f4/stm32f401/exti.rs.html#1-82` for me locally :)

Reply:
hm.. but how should it?

Reply:
This is handled by `rustdoc_html_handler`: https://github.com/rust-lang/docs.rs/blob/47ff0a349a38a1a21a22a5d0f84ae32bb3685dab/src/web/routes.rs#L143-L146
which already goes through match_version: https://github.com/rust-lang/docs.rs/blob/47ff0a349a38a1a21a22a5d0f84ae32bb3685dab/src/web/rustdoc.rs#L291-L297

Reply:
Oh wait I think i'm confused - source.rs is docs.rs' source view, not rustdoc's. Looks like this isn't currently supported either: https://docs.rs/crate/regex/*/source/Cargo.toml. I don't think fixing that needs to be part of this change.

Reply:
This will be fixed by #1350.